### PR TITLE
making operator frameworks guidelines clearer

### PR DESCRIPTION
**Pull request by Jenniferstrej**
@OmerKahani @thschue - do you think this is clearer?

### CODE REVIEW

File: operator-wg/whitepaper/05_Frameworks.md [hunk 1]
```diff
@@ -3,9 +3,25 @@
 ## Operator Frameworks for Kubernetes
 **(Current) Issue: https://github.com/cncf/sig-app-delivery/issues/40**
 
-[Framework_1](051_Framework_1.md) | Not even created | |
+## [Framework_1](051_Framework_1.md)
+
+- Add Short description of the framework
+- How does it work (in a Nutshell)?
+- What makes it unique?
```
Comment (nolar) [inline operator-wg/whitepaper/05_Frameworks.md:10]: This question about uniqueness seems ambiguous and highly opinionated. Maybe it is better to ask contributors to describe the delivered value or something else more specific?

Comment (Jenniferstrej) [reply operator-wg/whitepaper/05_Frameworks.md]: Applied a change based on your suggestion. Please take a look. Thanks!

**Comment by Jenniferstrej** [comment]
@nolar @thschue @OmerKahani - are you happy with the changes to the questions?

**Comment by nolar** [comment]
Yes, looks good to me. Thanks. But I cannot approve or resolve anything anyway.

**Review by thschue — APPROVED** [review]
lgtm

**Pull request merged**